lib/streamlit/elements/arrow.py

```diff
@@ -271,21 +271,46 @@ def dataframe(
     ) -> DeltaGenerator | DataframeState:
         """Display a dataframe as an interactive table.
 
-        This command works with dataframes from Pandas, PyArrow, Snowpark, and PySpark.
-        It can also display several other types that can be converted to dataframes,
-        e.g. numpy arrays, lists, sets and dictionaries.
+        This command works with a wide variety of collection-like and
+        dataframe-like object types.
 
         Parameters
         ----------
-        data : pandas.DataFrame, pandas.Series, pandas.Styler, pandas.Index, \
-            pyarrow.Table, numpy.ndarray, pyspark.sql.DataFrame, snowflake.snowpark.dataframe.DataFrame, \
-            snowflake.snowpark.table.Table, Iterable, dict, or None
+        data : dataframe-like, collection-like, or None
             The data to display.
 
+            Dataframe-like objects include dataframe and series objects from
+            popular libraries like Dask, Modin, Numpy, pandas, Polars, PyArrow,
```

Comment:
nit: should `pandas` be written uppercase here?

Reply:
I've looked on the pandas docs site and have not been able to find an instance of them capitalizing it? Even in plain-text sentences, it's lowercase: https://pandas.pydata.org/docs/getting_started/intro_tutorials/index.html

Reply:
Here's their branding site: https://pandas.pydata.org/about/citing.html
> When using the project name pandas, please use it in lower case, even at the beginning of a sentence.

Reply:
interesting, didn't know that :) maybe it looks less out of place if it is the first listed library.

Reply:
oh, but then it is not sorted anymore 🤔 yeah, probably its fine as is

Reply:
🤷🏻‍♀️ 😁